src/EditorFeatures/Core/CommandHandlers/QuickInfoCommandHandlerAndSourceProvider.QuickInfoSource.cs

```diff
@@ -22,26 +21,26 @@ public QuickInfoSource(QuickInfoCommandHandlerAndSourceProvider commandHandler,
                 _subjectBuffer = subjectBuffer;
             }
 
-            public void AugmentQuickInfoSession(IQuickInfoSession session, IList<object> quickInfoContent, out ITrackingSpan applicableToSpan)
+            public Task<QuickInfoItem> GetQuickInfoItemAsync(IAsyncQuickInfoSession session, CancellationToken cancellationToken)
             {
-                applicableToSpan = null;
-                if (quickInfoContent.Count != 0 ||
-                    session.Properties.TryGetProperty(QuickInfoUtilities.EventHookupKey, out object eventHookupValue))
+                if (session.Properties.TryGetProperty(QuickInfoUtilities.EventHookupKey, out object eventHookupValue))
```

Comment:
I understand that this is probably just here until the event hookup is moved over to the IToolTipService, but this is a pretty nasty hack. Just wanted to call it out in case you had missed it.